### ISSUE TITLE
Help @ Installation

### ISSUE THREAD

**Issue by hanshaze** [mame82/P4wnP1#243]
I have installed a fresh downloaded Raspbian Lite on my Pi Zero W, updated it, upgraded it, done Kernel Update so far and the on the go function i have also "unlocked"......

so my next step should be to install P4wnP1......
I installed git and downloaded the repo......#recursive

Then i used nano to customize the setup.cfg to this --->

![unbenannt](https://user-images.githubusercontent.com/17994050/40799061-ddb5816c-650c-11e8-9dd9-6f497967878f.PNG)

At this time my "/etc/network interfaces" looks like this --->

![unbenannt](https://user-images.githubusercontent.com/17994050/40799166-22dcdde4-650d-11e8-992e-7db200f4d6f6.PNG)

Now, the problem is, after a reboot i cant see a AP like described in setup.cfg....
I can connect to the Pi via 172.16.0.1 but i dont have internet connection anymore to install stuff....

At least i know, i have to switch setup.cfg to this to get "theoretically" internet access --->

![unbenannt](https://user-images.githubusercontent.com/17994050/40799424-d74c19c0-650d-11e8-89c4-e3b1f836e773.PNG)

I also customized the point of SSID/WPA in the setup.cfg file and the payload-file either....

Long story short...
WHY I DONT HAVE INTERNET ACCESS???

**Comment by FSEC-DALMASIAN** [comment]
did you under the wifi client part of the setup.cfg did you fill in your correct ssid and psk of the network that you want to connect to and did you make sure to change false to true?

**Comment by FSEC-DALMASIAN** [comment]
![capture](https://user-images.githubusercontent.com/31177296/40836576-8d475cf4-65da-11e8-9a85-b501b1cc72d5.PNG)

**Comment by harshil2105** [comment]
This whole attacking system is offline once installed. You have to make changes in the config file to connect to your know network connection like to your WiFi. Feel free to contact me if you face any more problems.
